index.html

```diff
@@ -181,7 +181,9 @@
 	<body onload="showdata();" style="background:black;color:white;font-family:monospace;">
 		<h1>ohai to pixelflut!</h1>
 		Real-time usage of the RGB color space quantized to eight sections on each axis:<br>
+		
 		<canvas id="datacanvas" width="512" height="512" style="border: none;"></canvas><br>
```

Comment:
shouldn't the size here be changed to the same as gl.viewportWidth/Height is set to?